integration/tx_relay_test.go

```diff
@@ -0,0 +1,116 @@
+package integration
+
+import (
+	"encoding/hex"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kaspanet/go-secp256k1"
+	"github.com/kaspanet/kaspad/txscript"
+	"github.com/kaspanet/kaspad/util"
+	"github.com/kaspanet/kaspad/wire"
+)
+
+func TestTxRelay(t *testing.T) {
+	payer, mediator, payee, teardown := standardSetup(t)
```

Comment:
I'm not sure I like the payer/payee terminology, because the node doesn't really pay to each other

Reply:
appHarness is not only node, it contains an address and privateKey